SubscribeIT/ngDesk
# Fixed catalogue bug

Closes #349 

Test case 1
Name :- Create catalogue
1. Login to ngdesk.
2. Goto -> company settings -> catalogue -> new.
3. Fill name , description and visible to.
4. Hit save.
5. Observe the error.

Test case 2
Name :- Edit catalogue
1. Login to ngdesk.
2. Goto -> company settings -> catalogue -> Click on above created catalogue.
3. Fill name , description and visible to.
4. Hit save.
5. Observe the error.

Screenshots :- 

![Screenshot from 2022-01-28 10-15-48](https://user-images.githubusercontent.com/89504119/151488832-eac2ca20-7b87-4a85-9fcb-385629e79e4f.png)
